src/app/code/community/Divante/VueStorefrontIndexer/Model/Index/Operations.php

```diff
@@ -178,7 +178,10 @@ public function createIndex($indexIdentifier, Store $store)
         $index = $this->initIndex($indexIdentifier, $store);
         $this->client->createIndex(
             $index->getName(),
-            ['index.mapping.total_fields.limit' => $this->indexSettings->getFieldsLimit()]
+            array_merge(
+                $this->indexSettings->getAnalysisConfig(), 
+                ['index.mapping.total_fields.limit' => $this->indexSettings->getFieldsLimit()]
```

Comment:
`index.mapping.total_fields.limit` -> please move this to `getAnalysisConfig` function

Reply:
Will fix it.

Reply:
maybe also rename to `getEsConfig` ? or sth. better.